build/webpack/plugins.js

```diff
@@ -29,25 +52,7 @@ const getPlugins = (app, grunt) => ([
             }),
         ]
         : [
-            function() {
-                this.plugin('watch-run', (watching, callback) => {
```

Comment:
replaced this part with a custom `WatchRunPlugin` since `this.plugin` no longer exists in webpack.